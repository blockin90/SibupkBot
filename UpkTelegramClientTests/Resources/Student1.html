<!DOCTYPE html>
<head>
	<link rel="stylesheet" type="text/css" href="TableStyle.css">
</head>
<body>

<div class="container container-top-header" align=center>
<h1>Расписание занятий</h1>
</div>
<table align="center">

		<tr>
		<td align="center"><b>СТУДЕНТАМ</b></td>
		<td colspan="3" align="center"><a href="index.php?mode=5">РАСПИСАНИЕ ЭКЗАМЕНАЦИОННОЙ СЕССИИ (ОЧНАЯ ФОРМА ОБУЧЕНИЯ)</a></td>
		</tr>
		<tr>
		<td align="center"><a href="index.php?mode=6">ОБЗОРНЫЕ ЛЕКЦИИ ПЕРЕД ГИА</a></td>
		<td align="center"><a href="index.php?mode=2">ПРЕПОДАВАТЕЛЯМ</a></td>
		<td align="center"><a href="index.php?mode=3">ПО АУДИТОРИЯМ</a></td>
		<td align="center"><a href="index.php?mode=4">ДЛЯ КАФЕДР</a></td>
		</tr>
		<tr>
		<td colspan="6">
		

				<p>Форма обучения: <b>очная</b></p>
				<p>Факультет: <b>Факультет экономики и управления</b></p>
				<p>Курс: <b>2 курс</b></p>
				<p>Группа: <b>ПИБ-01 (Прикладная информатика)</b></p>
				<p>Интервал: <b>14.02.2022 - 27.02.2022</b></p>
			<table width=100% border=0><tr><td align="left" valign="middle"><a href="index.php?mode=1" align="left">Вернуться назад</a></td><td align="right" valign="middle"><a target = "_blank" href="PrintTable.php?RangeNedel=14.02.2022 - 27.02.2022"><img src="Printer.png" alt="Печать" width="40"/></a></td></tr></table>
			<b>Группа ПИБ-01 (с 25 по 26 нед.)</b>
	<br><br><table class="table" width=100% cellspacing=0><tr>
		<th colspan=5 align=center><b><br>НЕЧЕТНАЯ НЕДЕЛЯ<br><br></b></th></tr>
		<tr>
		<th align=center><b>№ Пары</b></th>
		<th align=center><b>Дисциплина</b></th>
		<th align=center><b>Поток</b></th>
		<th align=center><b>Аудитория</b></th>
		<th align=center><b>Преподаватель</b></th>
		</tr><tr><th colspan=5><b>Понедельник (14.02.2022)</b></th></tr><tr>
			<td>3 (12:20-13:55)</td>
			<td>Базы данных (лаб)</td>
			<td>ПИБ-01.1</td>
			<td>а.227 УК 1 </td>
			<td>Доц., канд.техн.наук Астапчук В.А.</td>
			</tr><tr>
			<td>4 (14:05-15:40)</td>
			<td>Базы данных (лаб)</td>
			<td>ПИБ-01.1</td>
			<td>а.227 УК 1 </td>
			<td>Доц., канд.техн.наук Астапчук В.А.</td>
			</tr><tr>
			<td>5 (15:50-17:25)</td>
			<td>Дискретная математика (лек)</td>
			<td>ПИБ-01.1</td>
			<td>а.10 УК 2 </td>
			<td>Доц., канд.физ.-мат.наук Комиссаров В.В.</td>
			</tr><tr>
			<td>6 (17:35-19:10)</td>
			<td>Дискретная математика (с)</td>
			<td>ПИБ-01.1</td>
			<td>а.10 УК 2 </td>
			<td>Доц., канд.физ.-мат.наук Комиссаров В.В.</td>
			</tr><tr><th colspan=5><b>Вторник (15.02.2022)</b></th></tr><tr>
			<td>2 (10:15-11:50)</td>
			<td>Элективные дисциплины по физической культуре и спорту (Легкая атлетика/Волейбол/Баскетбол/Общая физическая подготовка) (с)</td>
			<td>ПИБ-01.1,  РБ-01.1</td>
			<td>а.Cпорт.зал УК 3 </td>
			<td>Ст. пр. Паулец Е.А.</td>
			</tr><tr>
			<td>3 (12:20-13:55)</td>
			<td>Информационные системы и технологии (лек)</td>
			<td>ПИБ-01.1</td>
			<td>а.215 УК 1 </td>
			<td>Зав. кафедрой, канд.техн.наук Тесля Н.Б.</td>
			</tr><tr>
			<td>4 (14:05-15:40)</td>
			<td>Теория вероятностей и математическая статистика (с)</td>
			<td>ПИБ-01.1</td>
			<td>а.10 УК 2 </td>
			<td>Доц., канд.физ.-мат.наук Комиссаров В.В.</td>
			</tr><tr>
			<td>5 (15:50-17:25)</td>
			<td>Теория вероятностей и математическая статистика (лек)</td>
			<td>ПИБ-01.1</td>
			<td>а.10 УК 2 </td>
			<td>Доц., канд.физ.-мат.наук Комиссаров В.В.</td>
			</tr><tr><th colspan=5><b>Среда (16.02.2022)</b></th></tr><tr>
			<td>3 (12:20-13:55)</td>
			<td>Базы данных (лек)</td>
			<td>ПИБ-01.1</td>
			<td>а.236 УК 1 </td>
			<td>Доц., канд.техн.наук Астапчук В.А.</td>
			</tr><tr><th colspan=5><b>Четверг (17.02.2022)</b></th></tr><tr>
			<td>4 (14:05-15:40)</td>
			<td>Теория вероятностей и математическая статистика (с)</td>
			<td>ПИБ-01.1</td>
			<td>а.10 УК 2 </td>
			<td>Доц., канд.физ.-мат.наук Комиссаров В.В.</td>
			</tr><tr>
			<td>5 (15:50-17:25)</td>
			<td>Информационные системы и технологии (лек)</td>
			<td>ПИБ-01.1</td>
			<td>а.317 УК 1 </td>
			<td>Зав. кафедрой, канд.техн.наук Тесля Н.Б.</td>
			</tr><tr>
			<td>6 (17:35-19:10)</td>
			<td>Факультативная дисциплина: Логика и теория аргументации (лек)</td>
			<td>ПИБ-01.1</td>
			<td>а.38 УК 2 </td>
			<td>Доц., канд.филос.наук Моргунов Г.В.</td>
			</tr><tr><th colspan=5><b>Пятница (18.02.2022)</b></th></tr><tr>
			<td>1 (08:30-10:05)</td>
			<td>Информационные системы и технологии (лаб)</td>
			<td>ПИБ-01.1</td>
			<td>а.204 УК 1 </td>
			<td>Зав. кафедрой, канд.техн.наук Тесля Н.Б.</td>
			</tr><tr><th colspan=5><b>Суббота (19.02.2022)</b></th></tr><tr>
			<td>2 (10:15-11:50)</td>
			<td>Программирование (лаб)</td>
			<td>ПИБ-01.1</td>
			<td>а.207 УК 1 </td>
			<td>преподаватель (практик) Блок И.Н.</td>
			</tr><tr>
			<td>3 (12:20-13:55)</td>
			<td>Операционные системы (лаб)</td>
			<td>ПИБ-01.1</td>
			<td>а.207 УК 1 </td>
			<td>преподаватель (практик) Блок И.Н.</td>
			</tr><tr>
			<td>4 (14:05-15:40)</td>
			<td>Операционные системы (лек)</td>
			<td>ПИБ-01.1</td>
			<td>а.134 УК 1 </td>
			<td>преподаватель (практик) Блок И.Н.</td>
			</tr><tr>
			<td>5 (15:50-17:25)</td>
			<td>Программирование (лек)</td>
			<td>ПИБ-01.1</td>
			<td>а.330 УК 1 </td>
			<td>преподаватель (практик) Блок И.Н.</td>
			</tr><tr>
		<th colspan=5 align=center><b><br>ЧЕТНАЯ НЕДЕЛЯ<br><br></b></th></tr>
		<tr>
		<th align=center><b>№ Пары</b></th>
		<th align=center><b>Дисциплина</b></th>
		<th align=center><b>Поток</b></th>
		<th align=center><b>Аудитория</b></th>
		<th align=center><b>Преподаватель</b></th>
		</tr><tr><th colspan=5><b>Понедельник (21.02.2022)</b></th></tr><tr>
			<td>3 (12:20-13:55)</td>
			<td>Базы данных (лаб)</td>
			<td>ПИБ-01.1</td>
			<td>а.227 УК 1 </td>
			<td>Доц., канд.техн.наук Астапчук В.А.</td>
			</tr><tr>
			<td>4 (14:05-15:40)</td>
			<td>Дискретная математика (лек)</td>
			<td>ПИБ-01.1</td>
			<td>а.13 УК 2 </td>
			<td>Доц., канд.физ.-мат.наук Комиссаров В.В.</td>
			</tr><tr>
			<td>5 (15:50-17:25)</td>
			<td>Дискретная математика (с)</td>
			<td>ПИБ-01.1</td>
			<td>а.13 УК 2 </td>
			<td>Доц., канд.физ.-мат.наук Комиссаров В.В.</td>
			</tr><tr><th colspan=5><b>Вторник (22.02.2022)</b></th></tr><tr>
			<td>2 (10:15-11:50)</td>
			<td>Элективные дисциплины по физической культуре и спорту (Легкая атлетика/Волейбол/Баскетбол/Общая физическая подготовка) (с)</td>
			<td>ПИБ-01.1,  РБ-01.1</td>
			<td>а.Cпорт.зал УК 3 </td>
			<td>Ст. пр. Паулец Е.А.</td>
			</tr><tr>
			<td>4 (14:05-15:40)</td>
			<td>Теория вероятностей и математическая статистика (с)</td>
			<td>ПИБ-01.1</td>
			<td>а.11 УК 2 </td>
			<td>Доц., канд.физ.-мат.наук Комиссаров В.В.</td>
			</tr><tr>
			<td>5 (15:50-17:25)</td>
			<td>Теория вероятностей и математическая статистика (лаб)</td>
			<td>ПИБ-01.1</td>
			<td>а.211 УК 1 </td>
			<td>Доц., канд.физ.-мат.наук Комиссаров В.В.</td>
			</tr><tr><th colspan=5><b>Среда (23.02.2022)</b></th></tr><tr>
			<td>1 (08:30-10:05)</td>
			<td>Нерабочий праздничный день</td>
			<td></td>
			<td></td>
			<td></td>
			</tr><tr><th colspan=5><b>Четверг (24.02.2022)</b></th></tr><tr>
			<td>4 (14:05-15:40)</td>
			<td>Теория вероятностей и математическая статистика (с)</td>
			<td>ПИБ-01.1</td>
			<td>а.012 УК 1 </td>
			<td>Доц., канд.физ.-мат.наук Комиссаров В.В.</td>
			</tr><tr>
			<td>5 (15:50-17:25)</td>
			<td>Теория вероятностей и математическая статистика (с)</td>
			<td>ПИБ-01.1</td>
			<td>а.023 УК 1 на 26нед</td>
			<td>Доц., канд.физ.-мат.наук Комиссаров В.В.</td>
			</tr><tr>
			<td>6 (17:35-19:10)</td>
			<td>Факультативная дисциплина: Логика и теория аргументации (лек)</td>
			<td>ПИБ-01.1</td>
			<td>а.38 УК 2 </td>
			<td>Доц., канд.филос.наук Моргунов Г.В.</td>
			</tr><tr><th colspan=5><b>Суббота (26.02.2022)</b></th></tr><tr>
			<td>2 (10:15-11:50)</td>
			<td>Программирование (лаб)</td>
			<td>ПИБ-01.1</td>
			<td>а.207 УК 1 </td>
			<td>преподаватель (практик) Блок И.Н.</td>
			</tr><tr>
			<td>3 (12:20-13:55)</td>
			<td>Операционные системы (лаб)</td>
			<td>ПИБ-01.1</td>
			<td>а.207 УК 1 </td>
			<td>преподаватель (практик) Блок И.Н.</td>
			</tr><tr>
			<td>4 (14:05-15:40)</td>
			<td>Операционные системы (лек)</td>
			<td>ПИБ-01.1</td>
			<td>а.330 УК 1 </td>
			<td>преподаватель (практик) Блок И.Н.</td>
			</tr><tr>
			<td>5 (15:50-17:25)</td>
			<td>Программирование (лек)</td>
			<td>ПИБ-01.1</td>
			<td>а.330 УК 1 </td>
			<td>преподаватель (практик) Блок И.Н.</td>
			</tr></table>
			</td></tr></table>
<h5><a href="http://sibupk.su">Вернуться на сайт СибУПК</a></h5>
</body></html>