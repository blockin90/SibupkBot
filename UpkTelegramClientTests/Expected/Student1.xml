<?xml version="1.0" encoding="utf-8" ?>
<ArrayOfStudentWorkDay xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
  <StudentWorkDay>
    <Date>2022-02-14T00:00:00</Date>
    <Lessons>
      <Lesson>
        <LessonNum>3</LessonNum>
        <Discipline>Базы данных</Discipline>
        <Group>ПИБ-01.1</Group>
        <Auditory>227 (1)</Auditory>
        <Online>false</Online>
        <LessonType>LabWork</LessonType>
        <TeacherName>Астапчук В.А.</TeacherName>
        <Date>2022-02-14T00:00:00</Date>
      </Lesson>
      <Lesson>
        <LessonNum>4</LessonNum>
        <Discipline>Базы данных</Discipline>
        <Group>ПИБ-01.1</Group>
        <Auditory>227 (1)</Auditory>
        <Online>false</Online>
        <LessonType>LabWork</LessonType>
        <TeacherName>Астапчук В.А.</TeacherName>
        <Date>2022-02-14T00:00:00</Date>
      </Lesson>
      <Lesson>
        <LessonNum>5</LessonNum>
        <Discipline>Дискретная математика</Discipline>
        <Group>ПИБ-01.1</Group>
        <Auditory>10 (2)</Auditory>
        <Online>false</Online>
        <LessonType>Lecture</LessonType>
        <TeacherName>Комиссаров В.В.</TeacherName>
        <Date>2022-02-14T00:00:00</Date>
      </Lesson>
      <Lesson>
        <LessonNum>6</LessonNum>
        <Discipline>Дискретная математика</Discipline>
        <Group>ПИБ-01.1</Group>
        <Auditory>10 (2)</Auditory>
        <Online>false</Online>
        <LessonType>Practical</LessonType>
        <TeacherName>Комиссаров В.В.</TeacherName>
        <Date>2022-02-14T00:00:00</Date>
      </Lesson>
    </Lessons>
  </StudentWorkDay>
  <StudentWorkDay>
    <Date>2022-02-15T00:00:00</Date>
    <Lessons>
      <Lesson>
        <LessonNum>2</LessonNum>
        <Discipline>Элективные дисциплины по физической культуре и спорту (Легкая атлетика/Волейбол/Баскетбол/Общая физическая подготовка) (с)</Discipline>
        <Group>ПИБ-01.1,  РБ-01.1</Group>
        <Auditory>а.Cпорт.зал УК 3 </Auditory>
        <Online>false</Online>
        <LessonType>Unknown</LessonType>
        <TeacherName>Паулец Е.А.</TeacherName>
        <Date>2022-02-15T00:00:00</Date>
      </Lesson>
      <Lesson>
        <LessonNum>3</LessonNum>
        <Discipline>Информационные системы и технологии</Discipline>
        <Group>ПИБ-01.1</Group>
        <Auditory>215 (1)</Auditory>
        <Online>false</Online>
        <LessonType>Lecture</LessonType>
        <TeacherName>Тесля Н.Б.</TeacherName>
        <Date>2022-02-15T00:00:00</Date>
      </Lesson>
      <Lesson>
        <LessonNum>4</LessonNum>
        <Discipline>Теория вероятностей и математическая статистика</Discipline>
        <Group>ПИБ-01.1</Group>
        <Auditory>10 (2)</Auditory>
        <Online>false</Online>
        <LessonType>Practical</LessonType>
        <TeacherName>Комиссаров В.В.</TeacherName>
        <Date>2022-02-15T00:00:00</Date>
      </Lesson>
      <Lesson>
        <LessonNum>5</LessonNum>
        <Discipline>Теория вероятностей и математическая статистика</Discipline>
        <Group>ПИБ-01.1</Group>
        <Auditory>10 (2)</Auditory>
        <Online>false</Online>
        <LessonType>Lecture</LessonType>
        <TeacherName>Комиссаров В.В.</TeacherName>
        <Date>2022-02-15T00:00:00</Date>
      </Lesson>
    </Lessons>
  </StudentWorkDay>
  <StudentWorkDay>
    <Date>2022-02-16T00:00:00</Date>
    <Lessons>
      <Lesson>
        <LessonNum>3</LessonNum>
        <Discipline>Базы данных</Discipline>
        <Group>ПИБ-01.1</Group>
        <Auditory>236 (1)</Auditory>
        <Online>false</Online>
        <LessonType>Lecture</LessonType>
        <TeacherName>Астапчук В.А.</TeacherName>
        <Date>2022-02-16T00:00:00</Date>
      </Lesson>
    </Lessons>
  </StudentWorkDay>
  <StudentWorkDay>
    <Date>2022-02-17T00:00:00</Date>
    <Lessons>
      <Lesson>
        <LessonNum>4</LessonNum>
        <Discipline>Теория вероятностей и математическая статистика</Discipline>
        <Group>ПИБ-01.1</Group>
        <Auditory>10 (2)</Auditory>
        <Online>false</Online>
        <LessonType>Practical</LessonType>
        <TeacherName>Комиссаров В.В.</TeacherName>
        <Date>2022-02-17T00:00:00</Date>
      </Lesson>
      <Lesson>
        <LessonNum>5</LessonNum>
        <Discipline>Информационные системы и технологии</Discipline>
        <Group>ПИБ-01.1</Group>
        <Auditory>317 (1)</Auditory>
        <Online>false</Online>
        <LessonType>Lecture</LessonType>
        <TeacherName>Тесля Н.Б.</TeacherName>
        <Date>2022-02-17T00:00:00</Date>
      </Lesson>
      <Lesson>
        <LessonNum>6</LessonNum>
        <Discipline>Факультативная дисциплина: Логика и теория аргументации</Discipline>
        <Group>ПИБ-01.1</Group>
        <Auditory>38 (2)</Auditory>
        <Online>false</Online>
        <LessonType>Lecture</LessonType>
        <TeacherName>Моргунов Г.В.</TeacherName>
        <Date>2022-02-17T00:00:00</Date>
      </Lesson>
    </Lessons>
  </StudentWorkDay>
  <StudentWorkDay>
    <Date>2022-02-18T00:00:00</Date>
    <Lessons>
      <Lesson>
        <LessonNum>1</LessonNum>
        <Discipline>Информационные системы и технологии</Discipline>
        <Group>ПИБ-01.1</Group>
        <Auditory>204 (1)</Auditory>
        <Online>false</Online>
        <LessonType>LabWork</LessonType>
        <TeacherName>Тесля Н.Б.</TeacherName>
        <Date>2022-02-18T00:00:00</Date>
      </Lesson>
    </Lessons>
  </StudentWorkDay>
  <StudentWorkDay>
    <Date>2022-02-19T00:00:00</Date>
    <Lessons>
      <Lesson>
        <LessonNum>2</LessonNum>
        <Discipline>Программирование</Discipline>
        <Group>ПИБ-01.1</Group>
        <Auditory>207 (1)</Auditory>
        <Online>false</Online>
        <LessonType>LabWork</LessonType>
        <TeacherName>Блок И.Н.</TeacherName>
        <Date>2022-02-19T00:00:00</Date>
      </Lesson>
      <Lesson>
        <LessonNum>3</LessonNum>
        <Discipline>Операционные системы</Discipline>
        <Group>ПИБ-01.1</Group>
        <Auditory>207 (1)</Auditory>
        <Online>false</Online>
        <LessonType>LabWork</LessonType>
        <TeacherName>Блок И.Н.</TeacherName>
        <Date>2022-02-19T00:00:00</Date>
      </Lesson>
      <Lesson>
        <LessonNum>4</LessonNum>
        <Discipline>Операционные системы</Discipline>
        <Group>ПИБ-01.1</Group>
        <Auditory>134 (1)</Auditory>
        <Online>false</Online>
        <LessonType>Lecture</LessonType>
        <TeacherName>Блок И.Н.</TeacherName>
        <Date>2022-02-19T00:00:00</Date>
      </Lesson>
      <Lesson>
        <LessonNum>5</LessonNum>
        <Discipline>Программирование</Discipline>
        <Group>ПИБ-01.1</Group>
        <Auditory>330 (1)</Auditory>
        <Online>false</Online>
        <LessonType>Lecture</LessonType>
        <TeacherName>Блок И.Н.</TeacherName>
        <Date>2022-02-19T00:00:00</Date>
      </Lesson>
    </Lessons>
  </StudentWorkDay>
  <StudentWorkDay>
    <Date>2022-02-21T00:00:00</Date>
    <Lessons>
      <Lesson>
        <LessonNum>3</LessonNum>
        <Discipline>Базы данных</Discipline>
        <Group>ПИБ-01.1</Group>
        <Auditory>227 (1)</Auditory>
        <Online>false</Online>
        <LessonType>LabWork</LessonType>
        <TeacherName>Астапчук В.А.</TeacherName>
        <Date>2022-02-21T00:00:00</Date>
      </Lesson>
      <Lesson>
        <LessonNum>4</LessonNum>
        <Discipline>Дискретная математика</Discipline>
        <Group>ПИБ-01.1</Group>
        <Auditory>13 (2)</Auditory>
        <Online>false</Online>
        <LessonType>Lecture</LessonType>
        <TeacherName>Комиссаров В.В.</TeacherName>
        <Date>2022-02-21T00:00:00</Date>
      </Lesson>
      <Lesson>
        <LessonNum>5</LessonNum>
        <Discipline>Дискретная математика</Discipline>
        <Group>ПИБ-01.1</Group>
        <Auditory>13 (2)</Auditory>
        <Online>false</Online>
        <LessonType>Practical</LessonType>
        <TeacherName>Комиссаров В.В.</TeacherName>
        <Date>2022-02-21T00:00:00</Date>
      </Lesson>
    </Lessons>
  </StudentWorkDay>
  <StudentWorkDay>
    <Date>2022-02-22T00:00:00</Date>
    <Lessons>
      <Lesson>
        <LessonNum>2</LessonNum>
        <Discipline>Элективные дисциплины по физической культуре и спорту (Легкая атлетика/Волейбол/Баскетбол/Общая физическая подготовка) (с)</Discipline>
        <Group>ПИБ-01.1,  РБ-01.1</Group>
        <Auditory>а.Cпорт.зал УК 3 </Auditory>
        <Online>false</Online>
        <LessonType>Unknown</LessonType>
        <TeacherName>Паулец Е.А.</TeacherName>
        <Date>2022-02-22T00:00:00</Date>
      </Lesson>
      <Lesson>
        <LessonNum>4</LessonNum>
        <Discipline>Теория вероятностей и математическая статистика</Discipline>
        <Group>ПИБ-01.1</Group>
        <Auditory>11 (2)</Auditory>
        <Online>false</Online>
        <LessonType>Practical</LessonType>
        <TeacherName>Комиссаров В.В.</TeacherName>
        <Date>2022-02-22T00:00:00</Date>
      </Lesson>
      <Lesson>
        <LessonNum>5</LessonNum>
        <Discipline>Теория вероятностей и математическая статистика</Discipline>
        <Group>ПИБ-01.1</Group>
        <Auditory>211 (1)</Auditory>
        <Online>false</Online>
        <LessonType>LabWork</LessonType>
        <TeacherName>Комиссаров В.В.</TeacherName>
        <Date>2022-02-22T00:00:00</Date>
      </Lesson>
    </Lessons>
  </StudentWorkDay>
  <StudentWorkDay>
    <Date>2022-02-23T00:00:00</Date>
    <Lessons>
      <Lesson>
        <LessonNum>1</LessonNum>
        <Discipline>Нерабочий праздничный день</Discipline>
        <Group />
        <Auditory />
        <Online>false</Online>
        <LessonType>Unknown</LessonType>
        <TeacherName />
        <Date>2022-02-23T00:00:00</Date>
      </Lesson>
    </Lessons>
  </StudentWorkDay>
  <StudentWorkDay>
    <Date>2022-02-24T00:00:00</Date>
    <Lessons>
      <Lesson>
        <LessonNum>4</LessonNum>
        <Discipline>Теория вероятностей и математическая статистика</Discipline>
        <Group>ПИБ-01.1</Group>
        <Auditory>012 (1)</Auditory>
        <Online>false</Online>
        <LessonType>Practical</LessonType>
        <TeacherName>Комиссаров В.В.</TeacherName>
        <Date>2022-02-24T00:00:00</Date>
      </Lesson>
      <Lesson>
        <LessonNum>5</LessonNum>
        <Discipline>Теория вероятностей и математическая статистика</Discipline>
        <Group>ПИБ-01.1</Group>
        <Auditory>023 (1)</Auditory>
        <Online>false</Online>
        <LessonType>Practical</LessonType>
        <TeacherName>Комиссаров В.В.</TeacherName>
        <Date>2022-02-24T00:00:00</Date>
      </Lesson>
      <Lesson>
        <LessonNum>6</LessonNum>
        <Discipline>Факультативная дисциплина: Логика и теория аргументации</Discipline>
        <Group>ПИБ-01.1</Group>
        <Auditory>38 (2)</Auditory>
        <Online>false</Online>
        <LessonType>Lecture</LessonType>
        <TeacherName>Моргунов Г.В.</TeacherName>
        <Date>2022-02-24T00:00:00</Date>
      </Lesson>
    </Lessons>
  </StudentWorkDay>
  <StudentWorkDay>
    <Date>2022-02-26T00:00:00</Date>
    <Lessons>
      <Lesson>
        <LessonNum>2</LessonNum>
        <Discipline>Программирование</Discipline>
        <Group>ПИБ-01.1</Group>
        <Auditory>207 (1)</Auditory>
        <Online>false</Online>
        <LessonType>LabWork</LessonType>
        <TeacherName>Блок И.Н.</TeacherName>
        <Date>2022-02-26T00:00:00</Date>
      </Lesson>
      <Lesson>
        <LessonNum>3</LessonNum>
        <Discipline>Операционные системы</Discipline>
        <Group>ПИБ-01.1</Group>
        <Auditory>207 (1)</Auditory>
        <Online>false</Online>
        <LessonType>LabWork</LessonType>
        <TeacherName>Блок И.Н.</TeacherName>
        <Date>2022-02-26T00:00:00</Date>
      </Lesson>
      <Lesson>
        <LessonNum>4</LessonNum>
        <Discipline>Операционные системы</Discipline>
        <Group>ПИБ-01.1</Group>
        <Auditory>330 (1)</Auditory>
        <Online>false</Online>
        <LessonType>Lecture</LessonType>
        <TeacherName>Блок И.Н.</TeacherName>
        <Date>2022-02-26T00:00:00</Date>
      </Lesson>
      <Lesson>
        <LessonNum>5</LessonNum>
        <Discipline>Программирование</Discipline>
        <Group>ПИБ-01.1</Group>
        <Auditory>330 (1)</Auditory>
        <Online>false</Online>
        <LessonType>Lecture</LessonType>
        <TeacherName>Блок И.Н.</TeacherName>
        <Date>2022-02-26T00:00:00</Date>
      </Lesson>
    </Lessons>
  </StudentWorkDay>
</ArrayOfStudentWorkDay>